pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -59,6 +59,9 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 	if err := localFeatureGate.SetFromMap(kc.FeatureGates); err != nil {
 		return err
 	}
+	if errs := localFeatureGate.Validate(); len(errs) > 0 {
```

Comment:
interesting... are there other places we're missing calls to Validate()? That was documented to be focused on emulated version, so I don't think node components were calling it... I see kube-proxy and kubelet calling SetFromMap in 7 places.

Also, should `SetFromMap` and `Set` refuse to make a change that would violate dependencies?